regions/io/ds9/tests/test_ds9_language.py

```diff
@@ -44,11 +46,24 @@ def test_fk5(filename):
     # print(actual)
     # 1/0
 
-    reference_file = get_pkg_data_filename('data/fk5_reference.reg')
+    if 'strip' in filename:
```

Comment:
In the diff I see that here in `regions/io/ds9/tests/test_ds9_language.py` the same additions appear 3 times. Can you move the code e.g. to a utility function to avoid the copy & paste?

Reply:
It's a pretty minor addition; a utility function would probably only save 1-2 lines of code and questionably improve maintainability.

Reply:
Note that you're not just duplicating this if / else 3 times, it's also this code block a few lines down that's copy & pasted tree times.
```
 # since metadata is not required to preserve order, we have to do a more
 +    # complex comparison
 +    desired_lines = [set(line.split()) for line in desired.split("\n")]
 +    actual_lines = [set(line.split()) for line in actual.split("\n")]
 +    for split_line in actual_lines:
 +        assert split_line in desired_lines
 +
 +    for split_line in desired_lines:
 +        assert split_line in actual_lines
```

Reply:
ah, right...

Reply:
yeah, that was wasteful.  New version is much more concise.